docs/proposals/Certificate Revocation Lists.md

```diff
@@ -0,0 +1,188 @@
+# CRL and CRL Cache Design
+
+## Intro
+
+Certificate validation is an essential step during signature validation. Currently Ratify supports checking for revoked certificates through OCSP supported by notation-go library. However, OCSP validation requires internet connection for each validation while CRL could be cached for better performance. As notary-project is adding the CRL support for notation signature validation, Ratify could utilize it.
```

Comment:
Hi @junczhu , clarification from a scenario level, CRL support is required for accurate validation result, and it cannot be disabled , correct?

Reply:
discussed in community, this can be controlled with trustPolicy